### ISSUE TITLE
README.md references setup.sh but this isn't on the Vagrant box

### ISSUE THREAD

**Issue by rmoff** [debezium/oracle-vagrant-box#10]
The README says: 

> To configure the database automatically run:
> 
> `cat setup.sh | docker exec -i dbz_oracle bash`

but `setup.sh` is part of this git repo, not present on the vagrant box itself. 

Should be mounted onto the image? Or the git repo cloned into the box as part of the setup process?

**Comment by rmoff** [comment]
One option: 

    curl https://raw.githubusercontent.com/debezium/oracle-vagrant-box/master/setup.sh | docker exec -i dbz_oracle bash

**Comment by jpechane** [comment]
Yes, I noticed it too. IMHO all three approaches are valid. Or `DOCKER_HOST` variable might be also solution.
